front/src/modules/ui/navigation/menu-item/components/MenuItem.tsx

```diff
@@ -48,7 +48,7 @@ export const MenuItem = ({
   return (
     <StyledHoverableMenuItemBase
       data-testid={testId ?? undefined}
-      onClick={handleMenuItemClick}
+      onClick={onClick ? handleMenuItemClick : undefined}
```

Comment:
let's make this change in handleMenuItemClick instead :)

Reply:
sure :+1: